.github/actions/on_device_tests/action.yaml

```diff
@@ -57,18 +57,21 @@ runs:
         GITHUB_PR_HEAD_USER_ID: ${{ github.event.pull_request.head.user.id }}
         GITHUB_COMMIT_AUTHOR_USERNAME: ${{ github.event.commits[0].author.username }}
         GITHUB_COMMIT_AUTHOR_EMAIL: ${{ github.event.commits[0].author.email }}
+        GITHUB_PR_NUMBER: ${{ github.event.pull_request.number }}
+        GITHUB_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
       run: |
         set -uxe
         python3 tools/on_device_tests_gateway_client.py \
           --token ${GITHUB_TOKEN} \
-          --change_id "${GITHUB_PR_HEAD_SHA:-$GITHUB_SHA}" \
+          --change_id ${GITHUB_PR_NUMBER:-postsubmit} \
```

Comment:
Is the default just the string "postsubmit"? Why don't we want to have the hash for postsubmit runs?

Reply:
We do, it's in a label below. Besides, the  PR number mapped better to change_id than the sha does.

Reply:
Are we considering nightlies as postsubmits?

Reply:
We would, yes, but I don't think it's an important distinction in this case as the field only ever ends up in logs anyway.